comment_scan.rb

```diff
@@ -200,25 +200,25 @@ def matches_bot(bot)
           tp_msg = [ #Generate tp line
             'tp'.center(6),
             tps.round(0).to_s.center(11),
-            "#{(tps*100/total).round(8)}%".center(14),
-            "#{(tps*100/Comment.where("tps >= ?", 1).count).round(8)}%".center(15),
-            "#{(tps*100/Comment.count).round(8)}%".center(18),
+            percent_str(tps, total).center(14),
+            percent_str(tps, Comment.where("tps >= ?", 1).count).center(15),
+	    percent_str(tps, Comment.count).center(18),
```

Comment:
Prolly should indent this the extra 2 spaces (and in `fp_msg` and in `total_msg`), but eh.

Reply:
@thesecretmaster guhhh my editor made it corrected my spaces to a tab. It'll be fixed in my next pull request, along with the editor settings >.>